README.adoc

```diff
@@ -49,9 +49,9 @@ the service reports `DOWN`, then it's unavailable. MicroProfile Health reports a
 service status at the endpoint and indicates the overall status as `UP` if all the services 
 are `UP`. A service orchestrator can then use the health statuses to make decisions.
 
-{kube} provides liveness and readiness probes that are used to check the health of your 
+{kube} provides startup, liveness, and readiness probes that are used to check the health of your 
 containers. These probes can check certain files in your containers, check a TCP socket, 
-or make HTTP requests. MicroProfile Health exposes readiness and liveness endpoints on 
+or make HTTP requests. MicroProfile Health exposes startup, liveness and readiness endpoints on 
```

Comment:
add a comma after "liveness"

...startup, liveness, and readiness endpoints....